JavaAssignments/src/vishal/StudentDetails.java

```diff
@@ -0,0 +1,27 @@
+package vishal;
+
+public class StudentDetails {
+	void printName(String name) {
+		System.out.println("Student Name is " + name);
+	}
+
+	void printRollNo(int rollNo) {
```

Comment:
printRollNo and printMarks method doing same job. 
These methods are called as redundant methods. 
You can have one method for ex. printData(int data) and another method printData(String data) ... these two methods having same name but different types of parameter .. its called overloading methods. we will learn it in upcoming session.